doc/howto/storage_pools.md

```diff
@@ -196,13 +211,33 @@ Storage pool my-pool pending on member vm03
 Storage pool my-pool created
 ```
 
-Also see {ref}`cluster-config-storage`.
+#### Create a remote storage pool
 
-```{note}
-For most storage drivers, the storage pools exist locally on each cluster member.
-That means that if you create a storage volume in a storage pool on one member, it will not be available on other cluster members.
+The following series of commands sets up a Ceph RBD storage pool with the name `my-remote-pool` and the on-disk name `my-osd` on three cluster members:
+As the configuration setting isn't member specific it has to be set when creating the actual storage pool:
 
-This behavior is different for Ceph-based storage pools (`ceph`, `cephfs` and `cephobject`) where each storage pool exists in one central location and therefore, all cluster members access the same storage pool with the same storage volumes.
+```{terminal}
+:input: lxc storage create my-remote-pool ceph --target=vm01
+Storage pool my-remote-pool pending on member vm01
+:input: lxc storage create my-remote-pool ceph --target=vm02
+Storage pool my-remote-pool pending on member vm02
+:input: lxc storage create my-remote-pool ceph --target=vm03
+Storage pool my-remote-pool pending on member vm03
+:input: lxc storage create my-remote-pool ceph ceph.osd.pool_name=my-osd
+Storage pool my-remote-pool created
+```
+
+Create a second storage pool `my-remote-pool2` using the Dell PowerFlex driver in SDC mode using the pool `sp1` in protection domain `pd1`:
```

Comment:
```suggestion
The following commands create a second storage pool `my-remote-pool2` using the Dell PowerFlex driver in SDC mode using the pool `sp1` in protection domain `pd1`:
```

Reply:
We could also be consistent with the examples in the other section and change all three examples to use "Create/Set up a pool ..." instead of "These commands create/set up ...".

Reply:
You mean not using the `{terminal}` block? I was thinking the same but decided against as it shows very cleanly how to to run the three `--target` commands one after another and then finally the actual pool creation command without too much text in between.

Or should we just indent the commands like in the other examples and prevent using the `{terminal}`?

Reply:
No sorry, I was referring to the wording only.
In the other examples section, we have instructions like "Create a directory pool" or "Use the existing directory", while here we are using "The following series of commands sets up" and "The following commands create".
So there's a bit of an inconsistency, but I don't see a problem with that.

Reply:
Ah ok I see. Added a PR to address the consistent naming https://github.com/canonical/lxd/pull/13827